File: AbcHealthCareApi/Entities/Medicine.cs
using System.ComponentModel.DataAnnotations;

namespace AbcHealthCareApi.Entities
{
    public class Medicine
    {
        [Key]
        public int IdMed { get; set; }
        public string NameMed { get; set; }
        public int PriceMed { get; set; }
        public string DescriptionMed { get; set; }
        [Required]
        public string Category { get; set; }
        public string SellerMed { get; set; }
        public string ImagePathMed { get; set; }
        public int Quantity { get; set; }
    }
}

File: abchealthcareappts/src/app/models/medicine.ts
import { StringDecoder } from "string_decoder";

export interface Medicine{
    idMed: number;
    nameMed: string;
    priceMed: number;
    descriptionMed?: any;
    category: string;
    sellerMed: string;
    imagePathMed?: any;
    quantity:number;
}